spring-addons-starter-rest/pom.xml

```diff
@@ -95,5 +95,21 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.wiremock.integrations</groupId>
+			<artifactId>wiremock-spring-boot</artifactId>
+			<version>${wiremock-springboot.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-test</artifactId>
+			<scope>test</scope>
+		</dependency>
 	</dependencies>
```

Comment:
To test with mocked OAuth2 identities, adding `spring-addons-oauth2-test` with `test` scope could be useful. To test Bearer forwarding, `@WithJwt` would configure the test case security context with an `Authentication` instance built from a `Jwt`.